Core/GDCore/Events/CodeGeneration/EventsCodeGenerator.cpp

```diff
@@ -728,6 +728,7 @@ gd::String EventsCodeGenerator::GenerateParameterCodes(
              metadata.GetType() == "tilesetResource" ||
              metadata.GetType() == "videoResource" ||
              metadata.GetType() == "model3DResource" ||
+             metadata.GetType() == "atlasResource" ||

```

Comment:
Can this type be also used for Sprite Atlas, e.g. TexturePacker format?
If not, be more specific e.g. `spineAtlasResource`